lib/g10_certification_test_kit/bulk_data_group_export.rb

```diff
@@ -0,0 +1,258 @@
+require_relative 'export_kick_off_performer'
+
+module G10CertificationTestKit
+  class BulkDataGroupExport < Inferno::TestGroup
+    title 'Group Compartment Export Tests'
+    description <<~DESCRIPTION
+      Verify that system level export on the Bulk Data server follow the Bulk Data Access Implementation Guide
+    DESCRIPTION
+
+    id :bulk_data_group_export
+
+    input :bearer_token
+    input :bulk_server_url,
+          title: 'Bulk Data FHIR URL',
+          description: 'The URL of the Bulk FHIR server.'
+    input :group_id,
+          title: 'Group ID',
+          description: 'The Group ID associated with the group of patients to be exported.',
+          default: '1'
+
+    output :requires_access_token, :status_output
+
+    fhir_client :bulk_server do
+      url :bulk_server_url
+    end
+
+    http_client :bulk_server do
+      url :bulk_server_url
+    end
+
+    # TODO: Create after implementing TLS Tester Class.
+    test do
+      title 'Bulk Data Server is secured by transport layer security'
+      description <<~DESCRIPTION
+        [§170.315(g)(10) Test Procedure](https://www.healthit.gov/test-method/standardized-api-patient-and-population-services) requires that
+        all exchanges described herein between a client and a server SHALL be secured using Transport Layer Security (TLS) Protocol Version 1.2 (RFC5246).
+      DESCRIPTION
+      # link 'http://hl7.org/fhir/uv/bulkdata/export/index.html#security-considerations'
+
+      run { pass }
+    end
+
+    test do
+      title 'Bulk Data Server declares support for Group export operation in CapabilityStatement'
+      description <<~DESCRIPTION
+        The Bulk Data Server SHALL declare support for Group/[id]/$export operation in its server CapabilityStatement
+      DESCRIPTION
+      # link 'http://hl7.org/fhir/uv/bulkdata/OperationDefinition-group-export.html'
+
+      run do
+        fhir_get_capability_statement(client: :bulk_server)
+        assert_response_status([200, 201])
+
+        assert_valid_json(request.response_body)
+        capability_statement = FHIR.from_contents(request.response_body)
+
+        definition = 'http://hl7.org/fhir/uv/bulkdata/OperationDefinition/group-export'
+
+        capability_statement.rest&.each do |rest|
+          groups = rest.resource&.select { |resource| resource.type == 'Group' }
+
+          pass if groups&.any? do |group|
+            group.operation&.any? do |operation|
+              if operation.definition.is_a? String
+                operation.definition == definition
+              else
+                operation.definition&.flatten&.include?(definition)
+              end
```

Comment:
`definition` is constrained `1..1` Cardinality-wise, so I don't think these checks are needed?